plutus-playground/plutus-playground-server/src/Playground/Usecases.hs

```diff
@@ -0,0 +1,18 @@
+{-# LANGUAGE TemplateHaskell #-}
+
+module Playground.Usecases where
+
+import           Data.ByteString (ByteString)
+import           Data.FileEmbed  (embedFile)
+
+vesting :: ByteString
+vesting = $(embedFile "./usecases/Vesting.hs")
```

Comment:
This is great, but why are we taking copies of these thing? I know Jann is planning to write simplified versions at some point, but until then shouldn't we use the ones from `plutus-use-cases`?

Reply:
Oh I see, they're quite different due to the pragmas now.

Reply:
they also use `mkFunction` to generate the schema, and that doesn't exist in `plutus-use-cases`